nex/register_common_secure_server_protocols.go

```diff
@@ -72,43 +71,41 @@ func stubBrowseMatchmakeSession(err error, packet nex.PacketInterface, callID ui
 	return rmcResponse, nil
 }
 
-func gameSpecificCanJoinMatchmakeSession(manager *commonglobals.MatchmakingManager, pid *types.PID, session *matchmakingtypes.MatchmakeSession) *nex.Error {
-	if !session.OpenParticipation.Value {
+func gameSpecificCanJoinMatchmakeSession(manager *commonglobals.MatchmakingManager, pid types.PID, session matchmakingtypes.MatchmakeSession) *nex.Error {
+	if !session.OpenParticipation {
 		return nex.NewError(nex.ResultCodes.RendezVous.PermissionDenied, "Gathering is not open to new participants")
 	}
 
 	isPublic := false
-	attrib, err := session.Attributes.Get(0)
-	if err == nil {
-		// * I wish this was a joke. top 8 bits are GameMode
-		// * This is the only difference between a public and a Friends match
-		isPublic = (attrib.Value & 0xFFFFFF) == 0x30881
-	}
+	attrib := session.Attributes[0]
+	// * I wish this was a joke. top 8 bits are GameMode
+	// * This is the only difference between a public and a Friends match
+	isPublic = (attrib & 0xFFFFFF) == 0x30881
```

Comment:
Can do `isPublic = (attrib & 0xFFFFFF) == 0x30881` and delete the `isPublic := false` line above if we want